### PR TITLE
[14.0][OU-ADD] l10n_fr: Nothing to do

### PR DESCRIPTION
**Pull request by legalsylvain**
Proposing this migration analysis. nothing to do.

However, I'm not totally sure regarding the renaming / replacement of all the ``account.tax.report.line``. (96 lines in V13 // 59 lines in V14)

As I personally don't use this model, another point of view would be welcome !

### CODE REVIEW

**Comment by legalsylvain** [comment]
CC : @alexis-via

**Comment by legalsylvain** [comment]
This PR is ready to merge.

File: docsource/modules130-140.rst [hunk 1]
```diff
@@ -298,7 +298,7 @@ Module coverage 13.0 -> 14.0
 +--------------------------------------------+-------------------------------------------------+
 |l10n_fi                                     |                                                 |
 +--------------------------------------------+-------------------------------------------------+
-|l10n_fr                                     |                                                 |
+|l10n_fr                                     | Done                                            |
```
Comment (pedrobaeza) [inline docsource/modules130-140.rst]:
```suggestion
|l10n_fr                                     | Nothing to do                                   |
```

**Pull request merged**